Library/Sender.ts

```diff
@@ -137,30 +137,32 @@ class Sender {
     }
 
     public saveOnCrash(payload: string) {
-        this._storeToDiskSync(payload);
+        if (this._enableOfflineMode) {
```

Comment:
Yes, we are already checking on L97 if enableOfflineMode is true, before starting to transmit stored files.